### PR TITLE
Link Prerender extension posts

### PR DESCRIPTION
**Pull request by tunetheweb**
Changes proposed in this pull request:

- Links the [Prerender pages](https://developer.chrome.com/blog/prerender-pages/) and [Extensions post on Prerender](https://developer.chrome.com/blog/extension-instantnav/)

### CODE REVIEW

**Review by jpmedley — CHANGES_REQUESTED** [review]
@tunetheweb,

Thank you for your submission. This are useful links to add to the document.

Please removed the 'updated' dates. We reserve those for technical updates ([handbook link](https://web.dev/handbook/yaml-front-matter/#updated). It helps lighten our workload when figuring out which documents need to be reviewed for possible update or rewrite.

Thanks,
Joe

File: site/en/blog/prerender-pages/index.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ description: |
 authors:
  - tunetheweb
 date: 2022-12-02
-#updated: 2022-12-02
+updated: 2022-12-14
```
Comment (tunetheweb) [inline site/en/blog/prerender-pages/index.md:10]:
```suggestion
#updated: 2022-12-02
```

**Comment by tunetheweb** [comment]
> Please removed the 'updated' dates. We reserve those for technical updates ([handbook link](https://web.dev/handbook/yaml-front-matter/#updated). It helps lighten our workload when figuring out which documents need to be reviewed for possible update or rewrite.

Done!

**Review by jpmedley — APPROVED** [review]
@tunetheweb,

Thank you again for your help.

Joe

**Pull request merged**